lib/config/common.ts

```diff
@@ -80,6 +80,7 @@ export interface RenovateAdminConfig {
 
   dockerImagePrefix?: string;
   dockerUser?: string;
+  dockerChildPrefix?: string;
```

Comment:
Is it really optional if it has a default value?

Reply:
That may be a misunderstanding on my part. 
If a default is set in the `definitions.ts` will the value always be present?
If so, then we can change it of course.

Reply:
```suggestion
  dockerChildPrefix: string;
```
🙃